ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -40,7 +40,7 @@ if [ ${LEASE_NUMBER} -ge 151 ] && [ ${LEASE_NUMBER} -lt 160 ]; then
 fi
 
 # For leases >= 200, run on the IBM Cloud vSphere 8 env
-if [ ${LEASE_NUMBER} -ge 200 ]; then
+if [ ${LEASE_NUMBER} -ge 200 ] && [ ${LEASE_NUMBER} -lt 220 ]; then
```

Comment:
The condition did not include the lease == 220, shall we include the 220 here?

Reply:
Currently
poc vSphere 8 have segments from 200 to 213
devqe vSphere have segments 221 to 234
I'm OK with setting ${LEASE_NUMBER} -lt 221